github3/issues/milestone.py

```diff
@@ -38,6 +38,8 @@ def _update_attributes(self, mile):
         self.due_on = self._strptime(mile.get('due_on'))
         #: datetime object representing when the milestone was updated.
         self.updated_at = self._strptime(mile.get('updated_at'))
+        #: string representing the milestone's ID.
+        self.id = mile._json_data['id']
```

Comment:
`mile` should be the dictionary object that has the original data. This should be

```py
self.id = mile.get('id')
```